test/ets_hash_ring_test.exs

```diff
@@ -285,25 +444,85 @@ defmodule ETSHashRingOperationsTest do
     assert Ring.find_nodes(HashRingETSTest.Empty, 1, 2) == {:error, :invalid_ring}
   end
 
+  test "find_previous_nodes" do
+    name = HashRingETSTest.Previous
+
+    {:ok, _pid} = Ring.start_link(name, nodes: @nodes, named: true)
+
+    {:ok, previous_generation} = Ring.get_ring_gen(name)
+
+    # Find primary and secondary for the currrent configuration
+    {:ok, [primary_a, secondary_a]} = Ring.find_nodes(name, 1, 2)
+
+    # Add multiple new nodes to the ring
+    {:ok, _} = Ring.add_nodes(name, ["d", "e", "f", "g", "h", "i", "j", "k"])
+
+    {:ok, current_generation} = Ring.get_ring_gen(name)
+
+    # Adding nodes atomically should increment the generation by 1
+    assert current_generation == previous_generation + 1
+
+    # Find primary and secondary for the new configuration
+    {:ok, [primary_b, secondary_b]} = Ring.find_nodes(name, 1, 2)
+
+    # Assert that the new configuration assigns different nodes to the target
+    assert primary_a != primary_b
+    assert secondary_a != secondary_b
+
+    {:ok, [primary_c, secondary_c]} = Ring.find_previous_nodes(name, 1, 2)
```

Comment:
nit: use self describing names?
```suggestion
    {:ok, [prev_primary, prev_secondary]} = Ring.find_previous_nodes(name, 1, 2)
```

Reply:
Adopted in https://github.com/discord/ex_hash_ring/pull/10/commits/173fbda6fff6a11075af73ad466eed548991667b